src/routes/pageRoutes.js

```diff
@@ -17,6 +18,9 @@ function PageRoutes() {
       <Navbar />
         <section className="content">
           <Switch>
+            <Route path="/signup" component={SignUp} />
```

Comment:
@Parathantl login and signup routes are already in the page routes. Please correct it